spring-boot/src/main/java/org/springframework/boot/web/server/Ssl.java

```diff
@@ -101,6 +101,31 @@
 	 */
 	private String protocol = "TLS";
 
+	/**
+	 * Disable SSL compression.
+	 */
+	private Boolean disableCompression;
```

Comment:
The `Ssl` configuration class is shared among all the embedded containers that we support (Jetty, Tomcat, and Undertow) which means that any configuration options that it offers have to be available across all three containers. Do you know if the 5 options you have proposed here are specific to Tomcat, or do Jetty and Undertow have equivalents for some/all of them?